src/content/contributing/release-process/_index.en.md

```diff
@@ -4,154 +4,209 @@ date: 2020-03-18T16:03:26+01:00
+simpler:
 
-To create lighthouse release artifacts follow the steps below.
+1) Navigate to the project's main releases page.
 
-### Build Lighthouse Agent and CoreDNS
+2) Click `Draft a new release`
 
-Assuming that you have an existing lighthouse git directory, run the following steps .
+3) Fill in the `Tag version` field with the new version.
 
-```bash
-cd lighthouse
-git stash
-git remote add upstream ssh://git@github.com/submariner-io/lighthouse
-git fetch -a -v -t upstream
-git checkout remotes/upstream/master -B master
-git tag -s -m "Globalnet Overlapping IP support RC0" v0.2.0-rc0
-git push upstream v0.2.0-rc0
-```
+4) Fill in the `Release title` field as appropriate. Typically this is just the version (sans the leading `v`) as a release
+   usually contains various changes. But if it's a targeted release then the `Release title` can reflect that.
+   
+5) Enter information to describe the release. This is optional for pre-releases but should be filled in for
+   a final release.
+   
+6) If this is not a final release, mark the checkbox `This is a pre-release`.
 
-A tagged release should appear [here](https://github.com/submariner-io/lighthouse/tags)
+7) Click `Publish release`.
 
-> https://github.com/submariner-io/lighthouse/tags
+For most projects a GitHub action job will be initiated to build release artifacts and publish to Quay. This will take
+several minutes. You can monitor the progress from the project's main page. A small yellow circle should be present to
+the right of the heading above the file listing which indicates it's in progress. You can click it to see details. When
+complete, it will change to either a green check mark or a red X.
 
-A build for v0.2.0-rc0 should start and appear under the "Active branches" section [here](https://travis-ci.com/github/submariner-io/lighthouse/branches)
+### Step 0: Create a `subctl` pre-release, optional
 
-> https://travis-ci.com/github/submariner-io/lighthouse/branches
+Since `subctl` is provided by `shipyard`'s dapper base image, if there are specific changes to `subctl` which other
+projects require, you will need to create a pre-release (for example `v0.5.0-pre0`) of the `submariner-operator` project: 
 
-For this example the build can be found [here](https://travis-ci.com/github/submariner-io/lighthouse/builds/153946391).
+1) Navigate to the [releases](https://github.com/submariner-io/submariner-operator/releases) page and create the release
+   with the new version.
 
-Verify that the build successfully completes as indicated by a green checkmark at the right. At this point the images tagged with 0.2.0-rc0 will be available on quay.io at:
+2) In the `shipyard` project, edit `package/Dockerfile.shipyard-dapper-base` and set `SUBCTL_VERSION` to the new version.
 
-> https://quay.io/repository/submariner/lighthouse-agent?tab=tags
-> https://quay.io/repository/submariner/lighthouse-coredns?tab=tags
+3) Commit the change, create a pull request and merge it.
```

Comment:
```suggestion
3) Commit the change, create a pull request and get it reviewed and merged.
```

Reply:
You might also want to instruct to label the PR with `test-projects` so it runs E2E on the consuming projects as well.